README.md

```diff
@@ -1,3 +1,18 @@
+<!--- BADGES: START --->
+[![GitHub - License](https://img.shields.io/github/license/danoneata/chalk?logo=github&style=flat&color=green)][#github-license]
+[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/chalk-diagrams?logo=pypi&style=flat&color=blue)][#pypi-package]
```

Comment:
Do we need all this stuff? Feels like noise.

Reply:
It is not necessary because it adds visual noise to the top of the readme and provides very little information. If someone needs to know these things it is trivial for them to look it up.